src/screens/ExchangeModal.js

```diff
@@ -377,7 +380,7 @@ export default function ExchangeModal({
     }
 
     const outputInUSD = outputPriceValue * outputAmount;
```

Comment:
this isn't a change from your PR, but just noticed that we should `multiply` using utilities, not like this (and we should also be updating `checkGasVsOutput` function to use utilities operations, not just assuming numbers